sdk/core/azure-core/CMakeLists.txt

```diff
@@ -132,7 +133,7 @@ set(
     src/operation_status.cpp
     src/strings.cpp
     src/uuid.cpp
-)
+    src/tracing/tracing.cpp)
```

Comment:
Nit: sort alpha ordering

Reply:
I'll make the change, but it's a pretty big change since nothing else in that list is alphabetical.